src/monaca/localPackageJSON.js

```diff
@@ -0,0 +1,158 @@
+
```

Comment:
The name of the file is misleading. The function is not related to the local package.json but to `project_info.json`